web/tilt_modules/uibutton/README.md

```diff
@@ -54,6 +54,17 @@ checkbox's state. By default, the variable will be set to the string `"true"` or
 | `true_string`  | `str`  | If not None, when the checkbox is checked, the environment variable will be set to this string instead of "true". |
 | `false_string` | `str`  | If not None, when the checkbox is checked, the environment variable will be set to this string instead of "false". |
 
+### `choice_input(name, label='', default='', choices=[])`
```

Comment:
i think this tilt_modules directory is vestigial, lemme remove it in another pr

Reply:
Should I be making a separate PR to https://github.com/tilt-dev/tilt-extensions/?

Reply:
ya, but you'll have to wait for a release

Reply:
Sounds good!